app/components/action-feed/template.hbs

```diff
@@ -8,11 +8,19 @@
     {{#each actionsList key='id' as |action|}}
         {{action-feed-entry action=action toDetail=toDetail}}
     {{/each}}
-    {{#if moreActions}}
+    {{#if (and moreActions (not loadingMore))}}
         <div class='see-more'>
```

Comment:
When I click on "See more", the "Send us your thoughts" banner jumps down while the loading indicator is displayed. Make sure the "See more" and the loading indicator div use the same amount of vertical space.